lib/api/model/pokemon_detail.dart

```diff
@@ -15,6 +17,8 @@ class PokemonDetail with _$PokemonDetail {
     @JsonKey(name: "abilities") List<Abilities>? abilities,
```

Comment:
haven't i explained before to avoid nullable collection, instead make non-nullable but make sure to always pass an empty collection? 🙂